graylog2-web-interface/src/views/components/widgets/UnknownWidget.tsx

```diff
@@ -85,7 +88,7 @@ const UnknownWidget = ({ config, type }: WidgetProps) => (
 );
 
 UnknownWidget.propTypes = {
-  config: PropTypes.object.isRequired,
+  config: PropTypes.instanceOf(AggregationWidgetConfig).isRequired,
```

Comment:
Are the propTypes even necessary with the `WidgetProps` type already existing?

Reply:
In general: yes. TS types are checked at compile time, prop types are checked at runtime. For this specific case (subclassing an interface) it is less important to have prop types to avoid repeating them for each subclass.